src/mono-runtimes/ProfileAssemblies.projitems

```diff
@@ -196,83 +196,4 @@
     <MonoProfileAssembly Include="System.Xml.Linq.dll" />
     <MonoProfileAssembly Include="System.Xml.Serialization.dll" />
   </ItemGroup>
-  <ItemGroup>
```

Comment:
Is it necessary to intermix the "mono archives" changes along with the mono/2018-10 bump changes?

I mean *sure*, mono/2018-10 has improved support for the mono archive, but I fear/suspect that the mono archive changes [are causing the NUnit tests to crash](
https://github.com/xamarin/xamarin-android/pull/2253#issuecomment-452954145), and thus a *smaller*, more focused PR may be more desirable (and then deal with the mono archive changes in a separate PR).